controllers/user.go

```diff
@@ -18,7 +19,7 @@ import (
 
 type keyedUserWithState struct {
 	*model.KeyedUser
-	State string
+	States []string
```

Comment:
Sure it makes sense. He could be invited to 2 challenges. Or he could be invited to one and coding another.